src/pages/gas.tsx

```diff
@@ -0,0 +1,493 @@
+                  <Translation id="page-gas-table-header-2" />
+                </Th>
+              </Tr>
+            </Thead>
+            <Tbody>
+              <Tr>
+                <Td>
+                  <Translation id="page-gas-table-item-1-transaction-type" />
+                </Td>
+                <Td>21,000</Td>
+              </Tr>
+              <Tr>
+                <Td>
+                  <Translation id="page-gas-table-item-2-transaction-type" />
+                </Td>
+                <Td>65,000</Td>
+              </Tr>
+              <Tr>
+                <Td>
+                  <Translation id="page-gas-table-item-3-transaction-type" />
+                </Td>
+                <Td>84,904</Td>
+              </Tr>
+              <Tr>
+                <Td>
+                  <Translation id="page-gas-table-item-4-transaction-type" />
+                </Td>
+                <Td>184,523</Td>
+              </Tr>
+            </Tbody>
+          </Table>
+        </Flex>
+      </Content>
+      <Content>
+        <H2 mt="0">
+          <Translation id="page-gas-faq-header" />
+        </H2>
+        {/* MaxWidth will be enforced by FAQ component once implemented */}
+        <Box maxWidth="832px">
+          <ExpandableCard title={t("page-gas-faq-question-1-q")}>
+            <Text>
+              <Translation id="page-gas-faq-question-1-a-1" />
+            </Text>
+            <Text>
+              <Translation id="page-gas-faq-question-1-a-2" />
+            </Text>
+          </ExpandableCard>
+          <ExpandableCard title={t("page-gas-faq-question-2-q")}>
+            <Text>
+              <Translation id="page-gas-faq-question-2-a-1" />
+            </Text>
+            <Link href="/eth/">
+              <Translation id="page-gas-faq-question-2-a-2" />
+            </Link>
+          </ExpandableCard>
+          <ExpandableCard title={t("page-gas-faq-question-3-q")}>
+            <Text>
+              <Translation id="page-gas-faq-question-3-a-1" />
+            </Text>
+            <Text>
+              <Translation id="page-gas-faq-question-3-a-2" />
+            </Text>
+          </ExpandableCard>
+        </Box>
+      </Content>
+      <Divider />
+      <Content>
+        <Flex wrap="wrap" mx={-4}>
+          <Box
+            as={Callout}
+            flex="1 1 416px"
+            minH="full"
+            image={whatIsEthereumImg}
+            titleKey="Use Layer 2"
+            alt=""
+            descriptionKey="Layer 2 extends Ethereum, reducing costs and increasing accessibility for decentralized applications."
+          >
+            <Box>
+              <ButtonLink to="/layer-2/">
+                <Translation id="page-gas-use-layer-2" />
+              </ButtonLink>
+            </Box>
+          </Box>
+          <Box
+            as={Callout}
+            flex="1 1 416px"
+            minH="full"
+            image={dogeComputerImg}
+            titleKey="page-community-explore-dapps-title"
```

Comment:
This `titleKey` is used with `t("common")` inside the `Callout` component... It requires the `page-community` namespace, but adding `page-community:` prefix doesn't work here; we may want/need to consider updating `Callout` to use the `Translation` component so a namespace prefix can be passed

Reply:
Same with `descriptionKey` below